package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/ADEHJKNTV/Yung.js.git"
   },
   "dependencies": {
-    "express": "^4.0.0"
+    "express": "^3.0.0"
```

Comment:
```suggestion
    "express": "^3.0.1"
```

Reply:
How dare u. We must stay pure with `3.0.0`